src/app/containers/FrontPageMain/__snapshots__/index.test.jsx.snap

```diff
@@ -1099,12 +1099,12 @@ exports[`FrontPageMain snapshots should render a pidgin frontpage correctly 1`]
             class="c2"
           >
             <section
-              aria-labelledby="Top-Stories"
+              aria-labelledby="Top-stories"
```

Comment:
not sure why this changed when updating snapshots...

Reply:
It's cause I'm now using all the index rules in the tests rather than just addIdsToItems.